website/docs/integrator-section/transferable-record/overview.md

```diff
@@ -4,43 +4,43 @@ title: Transferable Record Overview
+This tutorial builds on the knowledge from [verifiable document](/docs/integrator-section/verifiable-document/overview). If you have not completed the tutorial, please head back to [the previous section](/docs/integrator-section/verifiable-document/overview).
 
-By the end of this tutorial you would be able to create your "Electronic Bill of Lading" that is valid on any OA Viewer connected to the Ethereum goerli network.
+By the end of this tutorial, you will be able to create your "Electronic Bill of Lading" that is valid on any OA Viewer connected to the Ethereum sepolia network.
 
-With these knowledge you will be able to create transferable records according to your own business needs by:
+With the knowledge, you will be able to create transferable records according to your own business needs by:
 
-1. Changing the data structure of the document to suite your needs
+1. Changing the data structure of the document to fit your needs
 1. Changing the rendering of the document to reflect your document styles
-1. Changing the backend to use Ethereum main network
+1. Changing the backend to use the Ethereum main network
 
 ## Overview of Components
 
 ![Overview of Components](/docs/integrator-section/transferable-record/overview/overview.png)
 
-### Token Registry Smart Contract
+In the image above, we can see 3 different states of documents:
 
-The token registry smart contract is deployed by individual transferable records issuers such as the land title registry (for title deed) or shipping lines (for bill of lading). This smart contract replaces the document store smart contract in the previous section. similarly to document store contract, it also has it's identity bound to the issuer using DNS.
+1. An unissued document (`0xaaaa...aaaa`) will have an owner `0x0000...0000`
+1. An issued document (`0xbbbb...bbbb`) will have an owner which is either an Externally Owned Account (EOA) or a smart contract (ie `0x8888...8888`)
+1. A surrendered document (`0xcccc...cccc`) will have an owner which is the token registry address (ie `0x5555...5555`)
 
-The token registry stores the ownership state of the transferable records using a mapping from `document ID` to `owner`. The document ID is the target hash (and merkle root) of the individual OA document created. The owner will be either an externally owned account (EOA) or smart contract address.
+### Token Registry Smart Contract
 
-In the overview above, we can see 3 different states of documents:
+The token registry smart contract is deployed by individual transferable records issuers such as the land title registry (for Title Deed) or shipping lines (for Bill of Lading). This smart contract replaces the document store smart contract in [the previous section](/docs/integrator-section/verifiable-document/overview). Similar to document store contract, the token registry smart contract also has its identity bound to the issuer using DNS.
```

Comment:
the link from `the previous section` here sends the user back to `/verifiable-document/overview`. Since the context here is referring to the document store, should we link to `/verifiable-document/ethereum/document-store` instead?

Reply:
Link updated.